### ISSUE TITLE
Error message should have a dash instead of a period

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#58]
```
All .js 
and .css files in the jQuery Mobile directory /home/*USERNAME*/android- 
sdk/JQueryMobile/jquery-mobile/compiled must have the same version 
number of the form jquery.mobile.{version}.*

should be 

All .js 
and .css files in the jQuery Mobile directory /home/*USERNAME*/android- 
sdk/JQueryMobile/jquery-mobile/compiled must have the same version 
number of the form jquery.mobile-{version}.*

See 
https://groups.google.com/forum/#!topic/eclipse-plugin-for-phonegap/-3pC9pR7fTs


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 31 Dec 2011 at 4:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 6:59
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 1 Feb 2012 at 5:58
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
